numpy/_core/tests/test_stringdtype.py

```diff
@@ -718,6 +718,19 @@ def test_ufunc_add(dtype, string_list, other_strings, use_out):
             np.add(arr1, arr2)
 
 
+@pytest.mark.parametrize("use_out", [True, False])
+def test_ufunc_add_reduce(dtype, use_out):
```

Comment:
you're not using this `use_out` parameter

Reply:
Thanks, force pushed it away.

Reply:
Sorry I missed that!